classifiers/__init__.py

```diff
@@ -9,6 +9,10 @@
 
 from .lookup_lists import lookup_list
 
+from .query_check import (
```

Comment:
`question_type` instead of `query_check` ?

Reply:
Jup, changed it